### PR TITLE
Disabling dropout everywhere except optimize()

### PR DESCRIPTION
**Pull request by jorisvanmens**

### CODE REVIEW

File: qa_model.py [hunk 1]
```diff
@@ -509,8 +508,9 @@ def setup_embeddings(self):
         """
         with vs.variable_scope("embeddings"):
             embeddings = tf.constant(self.pretrained_embeddings, dtype=tf.float32)
-            self.question_embeddings_lookup = tf.nn.embedding_lookup(embeddings, self.question_placeholder)
-            self.context_embeddings_lookup = tf.nn.embedding_lookup(embeddings, self.context_placeholder)
+            embeddings_drop = tf.nn.dropout(embeddings, self.dropout_placeholder)
```
Comment (n2westman) [inline qa_model.py]: Do we want to drop the embeddings before the lookup?

Or do we want to drop them after (consistency with before)?

Comment (jorisvanmens) [reply qa_model.py]: Ah let's do after (I imagine the pre-lookup embeddings only get loaded once). Fixing..

**Comment by n2westman** [comment]
LGTM
